src/MusicalScore/Graphical/VexFlow/VexFlowMeasure.ts

```diff
@@ -965,12 +966,21 @@ export class VexFlowMeasure extends GraphicalMeasure {
         if (!this.rules.AutoBeamTabs && this.isTabMeasure) { // could also use an option tabBeams to disable beams there completely
             return;
         }
-        let notesToAutoBeam: StemmableNote[] = [];
```

Comment:
all the changes to autobeam generation are just to save note.NoteBeam for the auto-generated beam and also not create brackets for autogenerated tuplets if not necessary. (which needs to check note.NoteBeam)